src/main/java/frc/robot/RobotContainer.java

```diff
@@ -176,6 +160,10 @@ private void configureButtonBindings() {
                                          new ParallelCommandGroup(new IndexerOneCargo(), new RunIntakeCargo(true, 0))); 
             new JoystickButton(controlStick, XboxController.Button.kRightBumper.value).whenPressed(intakeDownFloat);
  
+            //intake reverse (use it only when it is needed.)
```

Comment:
This was not listed as part of the review, so you want to remap intake button

Reply:
We will finalize button mapping tonight with the drive team.